src/components/FindPark.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <section>
+    <h2>Let's Go Play!</h2>
+    <button @click="searchByLocation">Find a dog park near me!</button>
+    <h3>--Or--</h3>
+    <input
+      class="search-input"
+      type="text"
+      placeholder="Search by name, city or zip code"
+      ref="enteredValue"
+    />
+    <button class="search-button" @click="setSearchTerm">
+      Get Started - woof!
+    </button>
+    <p v-if="inputIsInvalid">Please enter a park name, city, or zip code</p>
+  </section>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      searchTerm: '',
+      inputIsInvalid: false,
+    };
+  },
+  methods: {
+    setSearchTerm() {
+      const enteredValue = this.$refs.enteredValue.value;
+      if (enteredValue === '') {
+        this.inputIsInvalid = true;
+      } else {
+        this.searchTerm = enteredValue;
```

Comment:
If `this.searchTerm` gets cleared in `this.search( )`, it might be a good idea to refactor this method by removing line 33 and substituting `this.search(this.searchTerm)` with `this.search(enteredValue).  No request for change - just brainstorming for when we look to refactor soon.  Nice work on this!